Headlines/src/Managers/ImageCacheManager.swift

```diff
@@ -16,28 +16,28 @@ class ImageCacheManager {
 
     var cachedImages = [String: UIImage]()
 
-    func image(forKey key: String) -> UIImage? {
+    func image(forKey key: String, completion: ((_ image: UIImage?) -> Void)?) {
 
         // Look for it on cache
         if let cachedImage = cachedImages[key] {
-            return cachedImage
+            completion?(cachedImage)
+            return
         }
 
-        // Look for it on disk
-        if let cachedImage = SDWebImageManager.shared().imageCache?.imageFromDiskCache(forKey: key) {
-            // Cache it on memory
-            cachedImages[key] = cachedImage
-            return cachedImage
-        }
+        SDWebImageManager.shared.imageCache.queryImage(forKey: key, options: [], context: nil) { [weak self] (image, _, _) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/3375a294d578861e49e3b4adad38548cd54ee1cc/Headlines/src/Managers/ImageCacheManager.swift#L27"></span>Line should be 120 characters or less: currently 125 characters<br />
<code>line_length</code> <code>ImageCacheManager.swift:27</code></td>
    </tr>
  </tbody>
</table>